examples/SharedMemory/SharedMemoryCommands.h

```diff
@@ -512,6 +519,16 @@ struct LoadSoftBodyArgs
 	double m_collisionMargin;
 	double m_initialPosition[3];
         double m_initialOrientation[4];
+        double m_springElasticStiffness;
+        double m_springDampingStiffness;
+        double m_corotatedMu;
+        double m_corotatedLambda;
+        double m_gravity[3];
```

Comment:
why does deformable needs its own gravity field? It is already in the regular setPhysicsEngineParameter?

Reply:
Removed extra gravity field.